pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
```

Comment:
Can these dependencies be simply defined in setup.py?

Reply:
We could put them in `setup.py` but it's a legacy way, that can break in some conditions. Basically `setup.py` can define dependencies fine, but cannot (very well) specify what is needed to run `setup.py` itself (i.e. build dependencies) . It's a chicken an egg problem. `pyproject.toml` was desined for that. 

See blog post in https://twitter.com/brettsky/status/1245207783240110080 for more details.

Reply:
But we can remove it, no one is enthusiastic about it.

Reply:
LGTM, let's use pyproject.toml